InsertionSortProj/InsertionSortTestProj/InsertionSortTests.cs

```diff
@@ -0,0 +1,39 @@
+using InsertionSort;
+using NUnit.Framework;
+
+namespace InsertionSortTestProj;
+
+[TestFixture]
+public class InsertionSortTests
+{
+    private readonly Random _rnd = new();
+    
+    [Test]
+    public void RandomArraySort()
+    {
+        CompareInsertionSortWithSystemSort(CreateIntArr());
+    }
+
+    [Test]
+    [TestCase(new [] { 6, 2, 9, 20, 130 }, TestName="PositiveNumbersSort" )]
```

Comment:
```suggestion
    [TestCase(new [] { 6, 2, 9, 20, 130 }, TestName = "PositiveNumbersSort" )]
```
Это ж не Питон какой-то :)